tutorial/katacoda-scenarios/flyte-install/step1.md

```diff
@@ -0,0 +1,6 @@
+Install flyte
+
+NOTE : Add Image and add description for the flyte component
```

Comment:
Let's add the following components. Mainly these two.

- FlyteAdmin
- Flytepropeller

You can copy description from here: https://github.com/lyft/flyteadmin and https://github.com/lyft/flytepropeller